Model/NodeType/Category.php

```diff
@@ -104,9 +113,33 @@ public function fetchData(array $nodes, $storeId)
         }
 
         $categoryUrls = $this->getResource()->fetchData($storeId, $categoryIds);
+        $categories = $this->getCategories($storeId, $categoryIds);
 
         $this->profiler->stop(__METHOD__);
 
-        return [$localNodes, $categoryUrls];
+        return [$localNodes, $categoryUrls, $categories];
+    }
+
+    /**
+     * @param int $storeId
```

Comment:
From what I see in `setStore` method definition, the method accepts all these types:

```php
@param int|string|\Magento\Store\Model\Store $store
```

So please update parameter name to be `$store` instead and add the accepted types to the annotations (not only `int`)

Reply:
Done